g11n-ws/modules/md-restful-api-i18n/src/main/java/com/vmware/vip/i18n/api/base/utils/CommonUtility.java

```diff
@@ -33,4 +33,24 @@ public static boolean checkParams(String[] categories, String... args){
 
         return true;
     }
+
+    /**
+     * Get categories according to CategoriesEnum
+     * @param scope
+     * @return
+     */
+    public static List<String> getCategories(String scope){
+        String[] categories = scope.split(",");
+        List<String> catList = new ArrayList<>();
+        for (String cat : categories) {
+            CategoriesEnum categoriesEnum = CategoriesEnum.getCategoriesEnumByText(cat);
+            if (categoriesEnum == null) {
+                catList.add(null);
```

Comment:
why need add null to catList

Reply:
This is for the convenience of parameter judgment in the checkParam method. Now I have migrated this part of the code (enum value acquisition) to checkParam method.